eureka2-server/build.gradle

```diff
@@ -23,6 +23,7 @@ dependencies {
     compile "org.slf4j:slf4j-simple:${slf4j_version}"
     compile "commons-cli:commons-cli:${commons_cli_version}"
 
+    compile "org.codehaus.jettison:jettison:${jettison_version}"
```

Comment:
karyon-admin brings in jettison 1.1, which does not work with serializing v1.Applications. (http://jettison.codehaus.org/)